extension/src/LanguageFunctions.ts

```diff
@@ -123,7 +116,7 @@ export async function findNextUnTranslatedText(
       }
     }
   }
-  return false;
+  return;
```

Comment:
Unnecessary, right? Or is this the difference between returning `void` or `undefined`?
Even with this return statement I believe that `TextDocumentMatch | undefined` is better than `TextDocumentMatch | void`

Reply:
Exactly. With undefined the compiler requires all code paths to return a value. With void it doesn't care.